src/builtins/core/datetime.rs

```diff
@@ -277,7 +278,36 @@ impl PlainDateTime {
         )
     }
 
+    /// Creates a new `DateTime` with an ISO 8601 calendar, constraining any arguments that into a valid range.
```

Comment:
Saw a couple of these throughout the PR, so just commenting on this one.

```suggestion
    /// Creates a new `DateTime` with an ISO 8601 calendar, constraining any arguments that may be invalid into a valid range.
```